tutorials/python-for-loops/index.mdx

```diff
@@ -0,0 +1,933 @@
+  else:
+    print(number, " is not greater than 5")
+  
+## Output:
+1 is not greater than 5
+10 is greater than 5
+9 is greater than 5
+4 is not greater than 5
+```
+
+<Message type="tip">
+
+Note that in the example above, we can print a variable and some text in the same string by separating them with commas and putting the text in quotation marks:
+
+```python
+print(variable_name, " text you want here", another_variable_name, " more text here")
+```
+
+</Message>
+
+### If/else exercises
+
+1. Create a loop that iterates over the list `[1, 10, 9, 4]` and checks whether each number is greater than 3. The output should look like this:
+
+    ```
+    1 is not greater than 3
+    10 is greater than 3
+    9 is greater than 3
+    4 is greater than 3
+    ```
+
+    Answer [below](#ifelse-exercises:-answers).
+
+3. Create a loop that iterates over the list `["France", "Japan", "the USA"]` and searches for "the USA" in the list. The output should look like this:
+
+    ```
+    This country is not the USA
+    This country is not the USA
+    This country is the USA
+    ```
+
+    Answer [below](#ifelse-exercises:-answers).
+
+3. Create a loop that iterates over the list `[-1, 2, 3, 0, -4]` and checks whether each number is positive or negative. The output should look like this:
+
+    ```
+    -1 is a negative number
+    2 is a positive number
+    3 is a positive number
+    0 is neither positive nor negative
+    -4 is a negative number
+    ```
+
+    Answer [below](#ifelse-exercises:-answers).
+
+## For loops with a range() function
+
+As we saw in an earlier example we can use loops with Python's built-in `range()` function to state exactly how many times we want the loop to iterate.
+
+This function can be used in a number of ways:
+
+With just one parameter, the range starts at a default of 0 and finishes at (not including) the specified number:
+
+```python
+for x in range(3):
+  print(x)
+  
+## Output:
+0
+1
+2
+```
+
+With two parameters, we set the start and finish numbers of the range:
+
+```python
+for x in range(2, 6):
+  print(x)
+  
+## Output:
+2
+3
+4
+5
+```
+
+With three parameters, we set the start and finish numbers of the range, and the **step** value (the interval between each iteration):
+
+```python
+for x in range(2, 10, 2):
+  print(x)
+  
+## Output:
+2
+4
+6
+8
+```
+
+### Range exercises
+
+1. Create a loop that iterates over a range and produces the following output:
+
+    ```
+    5
+    6
+    7
+    8
+    9
+    ```
+
+    Answer [below](#range-exercises:-answers).
+  
+2. Create a loop that iterates over a range and produces the following output:
+
+    ```
+    0
+    25
+    50
+    75
+    ```
+
+    Answer [below](#range-exercises:-answers).
+
+3. Create code that includes a loop that iterates over a range, and produces the following output:
+
+    ```
+    Counting down...
+    5
+    4
+    3
+    2
+    1
+    Go!
+    ```
+
+    Answer [below](#range-exercises:-answers).
+
+## Nested for-loops
```

Comment:
```suggestion
## Nested for-loops
```

for loops or for-loops? As someone who isn't familiar with Python, I'd lean towards `for-loops` but it seems `for loops` is more common (and preferable SEO wise?)

Reply:
Hmm, I checked and thought that for-loops seemed more common, but I'll have another look